google/cloud/bigtable/examples/run_examples_utils.sh

```diff
@@ -235,8 +235,6 @@ function run_all_table_admin_examples {
       "${project_id}" "${INSTANCE}" "${TABLE}" "nested-family"
   run_example ./table_admin_snippets delete-column-family \
       "${project_id}" "${INSTANCE}" "${TABLE}" "nested-family"
-  run_example ./table_admin_snippets check-family-exists \
```

Comment:
Perhaps you keep this and expect it to fail?

Reply:
Done.